resources/lang/es/auth.php

```diff
@@ -21,9 +21,12 @@
     'language' => 'Idioma',
     'login' => 'Ingresar',
     'register' => 'Registrate',
-    'notMemberYet' => '¿Todavía no tienes una cuenta?',
+    'notMemberYet' => 'Todavía no tenés una cuenta',
```

Comment:
why was this changed? tenes is a form of spanish only used in argentina. and its stupid